package.json

```diff
@@ -9,28 +9,28 @@
     "diagnostics": "^1.0.1",
     "express": "^4.13.3",
     "lodash.merge": "^4.6.2",
-    "nconf": "^0.8.2",
+    "nconf": "^0.11.3",
+    "snyk": "^1.316.1",
     "understudy": "^4.1.0",
-    "winston": "^3.1.0",
-    "snyk": "^1.316.1"
+    "winston": "^3.1.0"
   },
   "devDependencies": {
     "chai": "^3.4.1",
-    "eslint": "^4.19.1",
-    "eslint-config-godaddy": "^2.1.0",
-    "eslint-plugin-json": "^1.2.1",
-    "eslint-plugin-mocha": "^4.12.1",
-    "godaddy-test-tools": "^13.0.0",
```

Comment:
I've been removing usage of this because the `gulp` maintainer that also owns vulnerable packages like `glob-parent` refuses to fix reported vulnerabilities because he says they're all false positives. However, this affects applications with security policies that check vulnerability reports, so I think we should divest from gulp. It's nice that `godaddy-test-tools` provides us an all-in-one testing package, but it's more of a convenience than something essential.